pkg/specgen/generate/kube/volume.go

```diff
@@ -57,7 +57,7 @@ func VolumeFromHostPath(hostPath *v1.HostPathVolumeSource) (*KubeVolume, error)
 		switch *hostPath.Type {
 		case v1.HostPathDirectoryOrCreate:
 			if _, err := os.Stat(hostPath.Path); os.IsNotExist(err) {
```

Comment:
Remove the Stat here, MkdirAll will already calls stat anyway

Reply:
Done (and new test has been added)